### PR TITLE
Remove Commons Lang dependency

### PR DESCRIPTION
**Pull request by stefan-kolb**
Cleanup. Remove dependencies.

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/push/PushToApplicationAction.java [hunk 1]
```diff
@@ -90,10 +90,11 @@ private static String getKeyString(List<BibEntry> entries) {
 
     @Override
     public void execute() {
-        // If required, check that all entries have citation keys defined:
+        // If required, check that all entries have citation keys defined
         if (application.requiresCitationKeys()) {
             for (BibEntry entry : stateManager.getSelectedEntries()) {
-                if (StringUtil.isBlank(entry.getCitationKey())) {
+                Optional<String> citationKey = entry.getCitationKey();
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/push/PushToApplicationAction.java:96]: Use optional orElseGet?
https://www.baeldung.com/java-optional-or-else-vs-or-else-get

**Review by tobiasdiez — COMMENTED** [review]
I don't quite understand these changes. It looks like you inlined the two helper methods from `StringUtil` but why is not quite clear to me.

**Comment by Siedlerchr** [comment]
The apache commons long StrinUtil is blank does also some more checks than just simple whitespace

**Comment by stefan-kolb** [comment]
Thanks for your reviews guys, but this is really draftish ;-)

**Comment by koppor** [comment]
Hint: https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/architecture/ApacheCommonsLang3Allowed.java

**Comment by calixtus** [comment]
Devcall: since there seems to be no activity here, we're freezing this PR now.
Reopening shouldn't be a problem.

**Pull request closed without merging**